dependencies.yaml

```diff
@@ -459,43 +514,68 @@ dependencies:
           - ucxx-tests==0.46.*,>=0.0.0a0
   depends_on_ucx_build:
     common:
-      - output_types: conda
-        packages:
-          - ucx==1.17.0
       - output_types: requirements
         packages:
           # pip recognizes the index as a global option for the requirements.txt file
           - --extra-index-url=https://pypi.nvidia.com
           - --extra-index-url=https://pypi.anaconda.org/rapidsai-wheels-nightly/simple
     specific:
+      - output_types: [conda]
+        matrices:
+          - matrix:
+              cuda: "12.*"
+            packages:
+              - ucx==1.17.0
+          # build against 1.19.0 on CUDA 13 (and make that the default when 'cuda' filter is not provided)
+          - matrix:
+            packages:
+              - ucx==1.19.0
       - output_types: [requirements, pyproject]
         matrices:
           - matrix:
               cuda: "12.*"
               cuda_suffixed: "true"
             packages:
               - libucx-cu12>=1.17.0,<1.17.1a0
+          - matrix:
+              cuda: "13.*"
+              cuda_suffixed: "true"
+            packages:
+              - libucx-cu13>=1.19.0,<1.19.1a0
           - matrix:
             packages:
-              - libucx>=1.17.0,<1.17.1a0
+              - libucx>=1.19.0,<1.19.1a0
```

Comment:
I'm not sure if I'm misunderstanding something, but it looks like we're limiting to those specific libucx versions here, no? I.e., only `libucx-cu12==1.17.0` and `libucx-cu13==1.19.0` will be installable because of the tight upper-bound. I think we want `>=1.17.0,<1.20`/`>=1.19.0,<1.20` (after fixing the syntax which I'm never certain of), no?